packages/components/Atoms/Icon/Icon.vue

```diff
@@ -87,6 +88,22 @@ export default {
 
       return icons[this.symbol]
     },
+    iconText () {
+      if (!this.icon && !this.legacyIcon) return ''
+      if (this.symbol === 'external_link') {
+        return 'External Link'
+      } else {
+        return ''
+      }
+    },
+    ariaHidden () {
+      if (!this.icon && !this.legacyIcon) return 'TRUE'
+      if (this.symbol === 'external_link') {
+        return 'FALSE'
+      } else {
+        return 'TRUE'
+      }
+    },
```

Comment:
Hey @mudrasamey, had another look, could you maybe combine those 2 methods and set those 2 properties at the same time? The checks are identical in both, so it's mostly duplication. 

Can you also have a look through the codebase for other examples where `aria-hidden` is used and see if it's all caps and if not use whatever casing is already used. Thanks!

Reply:
@elNico I have removed the duplicate method and replaced the aria-hidden to lowercase to match rest of the implementations

Reply:
@elNico I see this has been implemented - can you look at this again?